R/Augment_Snapshot.R

```diff
@@ -72,6 +74,58 @@ Augment_Snapshot <- function(
       })
   }
 
+  if (bAppendLongitudinalResults) {
+    snapshot_date <- list.files(cPath)
+    snapshot_directory_names <- paste0(cPath, snapshot_date)
```

Comment:
for this one I added logic to check if "/" was at the end of cPath instead, because some people may make `cPath = "~/gsm/inst/snapshots"` and others may use `cPath = "~/gsm/inst//snapshots/"`